form-builder/src/main/java/com/dsc/form_builder/Validators.kt

```diff
@@ -76,6 +76,12 @@ sealed interface Validators {
 
     /**
      * This is validator gives you the option to provide your own implementation of the validator. You can pass in a custom function to validate the field value.
+     *
+     * Example:
```

Comment:
You can add code snippets in the suggestion to provide a clearer example in this case. 

```suggestion
     * Example: check if a string contains the word hello
     * ```kt
     * Validators.Custom(
     *     message = "value must have hello"
     *     function = { it.contains("hello") }
     * )
     * ```
```


This is how it looks in the IDE:
|  KDoc | Hint  |
|---|---|
| <img width="732" alt="Screenshot 2023-05-02 at 07 57 51" src="https://user-images.githubusercontent.com/47350130/235590398-58440d6d-288f-4d86-86db-f7c990566b23.png">  | <img width="622" alt="Screenshot 2023-05-02 at 07 58 22" src="https://user-images.githubusercontent.com/47350130/235590452-31cb599c-b986-4a19-9505-156767a52a89.png">  |

Reply:
I have added the suggested implementation on this [PR #58](https://github.com/jkuatdsc/form-builder/pull/58). LGTM